app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(
+            self,
+            visitor: dict
+    ) -> str | VaccineError | OutdatedVaccineError:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("visitor is not vaccinated")
+        if datetime.date.today() > visitor["vaccine"]["expiration_date"]:
```

Comment:
It is good practice to place in the first position for comparison an incoming value and at the second position a value it is compared against. Change the position of arguments that you compare on this line.